src/common/delivery/deploy/local/jar/executableJarDeployer.ts

```diff
@@ -46,12 +46,12 @@ export function executableJarDeployer(opts: LocalDeployerOptions): ArtifactDeplo
     });
 }
 
-class ExecutableJarDeployer implements ArtifactDeployer<ManagedDeploymentTargetInfo> {
+class ExecutableJarDeployer implements ArtifactDeployer<ManagedDeploymentTargetInfo, Deployment> {
```

Comment:
Not entirely sure why deployers implement `Deployment`...probably need to discuss this to understand

Reply:
ArtifactDeployer now takes a second type of generic for Deployment. In other cases, like cf, it makes more sense because there are specific attributes (like an id that can be used for undeploying). In this case I just used Deployment to keep the contract the same for now until there is need of a specific impl.